ui/src/Components/notification/index.jsx

```diff
@@ -0,0 +1,81 @@
+import { useState } from "react";
+import { TYPETOCOLOR } from "../../Constant/constant";
+
+/**
+ * a pubsub kind of implementation, basically it will save a function in closure with varaible called "fn"
+ * when i use a function called subcribe and pass it with the function i want to save
+ * then when i will call publish function with "data" parameter it will call the function present in "fn"
+ * with the "data" as a arguement
+ */
+export const NotificationQue = () => {
+  let fn = () => {};
+  const subscribe = (callback) => {
+    fn = callback;
+  };
+  const publish = (data) => {
+    return fn(data);
```

Comment:
We should have the View only one component. That component would be reusable across projects. As of now, component is tightly bundled with the logic also. Break the components.